source/precalculus/source/08-TE/01.ptx

```diff
@@ -12,6 +12,1187 @@
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <me>\frac{a^2}{b^2}+\frac{b^2}{b^2}=\frac{c^2}{b^2}</me>, which simplifies to
+            <me>
+              \frac{a^2}{b^2}+1=\frac{c^2}{b^2}
+            </me>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            How could we rewrite the equation we got in part (a) so that each term with a fraction can be represented as a fraction squared?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m> \left( \frac{a}{b} \right) ^2 + 1 =\left( \frac{c}{b} \right) ^2</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Let's now try and think of the relationships of the angle and sides (i.e., our trigonometric ratios). Replace <m>a</m>, <m>b</m>, and <m>c</m> in the equation we got in part (b) with the relationship each side has with <m>\theta</m> (i.e., opposite, adjacent, hypotenuse).
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Replace the <m>a</m>, <m>b</m>, and <m>c</m> in the equation <me> \left( \frac{a}{b} \right) ^2 + 1 = \left( \frac{c}{b} \right) ^2</me> will yield <me>\left( \frac{\text{adjacent}}{\text{opposite}} \right) ^2 + 1 = \left( \frac{\text{hypotenuse}}{\text{opposite}} \right) ^2</me> 
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Recall the following trigonometric functions:
+            <ul cols="2"> 
+              <li><m>\sin\theta=\frac{\text{opposite}}{\text{hypotenuse}}
+              </m></li>
+              <li><m>
+                \csc\theta=\frac{\text{hypotenuse}}{\text{opposite}}
+              </m></li>
+              <li><m>
+                \cos\theta=\frac{\text{adjacent}}{\text{hypotenuse}}
+              </m></li>
+              <li><m>
+                \sec\theta=\frac{\text{hypotenuse}}{\text{adjacent}}
+              </m></li>
+              <li><m>
+                \tan\theta=\frac{\text{opposite}}{\text{adjacent}}
+              </m></li>
+              <li><m>
+                \cot\theta=\frac{\text{adjacent}}{\text{opposite}}
+              </m></li>
+            </ul>
+            Using these trigonometric ratios, how can we rewrite the equation in part (c)?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\cot^2\theta + 1 = \csc^2\theta</m>
+          </p>
+        </answer>
+      </task>
+    </activity>
+
+    <remark>
+      In <xref ref="deriving-pythagorean-identity-3"/>, we derived the third Pythagorean Identity: <m>\csc^2\theta-\cot^2\theta=1</m>. Note that we can rewrite what we got in the previous activity <m> \left( \cot^2\theta + 1 = \csc^2\theta \right)</m> to get to this form. 
+    </remark> 
+
+    <definition xml:id="def-pythagorean-identities">
```

Comment:
I think this (and others below) are really theorems rather than definitions.

Reply:
How do I make it so that the text is not all italic? I want the "note" part to be regular text.

Reply:
The styling of notes vs theorems vs definitions, etc, is not an author toggle and cannot be done for just one instance. It's best to choose a tag that matches is the semantic purpose of the block, and if you don't like the styling we need to discuss changing it globally across the books.